tools/ansible/tasks/wsu/test/e2e/wsu_test.go

```diff
@@ -36,8 +36,9 @@ var (
 
 	// The CI-operator uses AWS region `us-east-1` which has the corresponding image ID: ami-0b8d82dea356226d3 for
 	// Microsoft Windows Server 2019 Base with Containers.
-	imageID      = "ami-0b8d82dea356226d3"
-	instanceType = "m4.large"
+	imageID = "ami-0b8d82dea356226d3"
+	// Using this instance type, as the Windows hybrid overlay currently does not work on intel processors
```

Comment:
It has nothing to do with the process but a bug in the Intel 82599 network driver. Please state that here.

Reply:
This needs to be a separate PR as I want this clearly called out and not buried inside a PR that has nothing to do with this change.

Reply:
Moved to https://github.com/openshift/windows-machine-config-operator/pull/106